### DISCUSSION TITLE
ERROR ITMS-90179

### DISCUSSION THREAD

**Discussion by zhoujun951236** [fastlane/fastlane#18678] [Q&A]
ERROR ITMS-90179: "Invalid Code Signing. The executable 'Payload/pluto.app/Frameworks/AFNetworking.framework/AFNetworking' must be signed with the certificate that is contained in the provisioning profile."

ERROR ITMS-90179: "Invalid Code Signing. The executable 'Payload/pluto.app/Frameworks/Alamofire.framework/Alamofire' must be signed with the certificate that is contained in the provisioning profile."

ERROR ITMS-90179: "Invalid Code Signing. The executable 'Payload/pluto.app/PlugIns/notificationServiceExtension.appex/notificationServiceExtension' must be signed with the certificate that is contained in the provisioning profile."

ERROR ITMS-90179: "Invalid Code Signing. The executable 'Payload/pluto.app/PlugIns/plutoTodayWidget.appex/plutoTodayWidget' must be signed with the certificate that is contained in the provisioning profile."

ERROR ITMS-90179: "Invalid Code Signing. The executable 'Payload/pluto.app/pluto' must be signed with the certificate that is contained in the provisioning profile."

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 11.0.1                                                         |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.24.3 (Apple Git-128)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 11.0.1 (20B50)                                           |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 12.2                                                           |

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.182.0 | ✅ Up-To-Date  |

resign(ipa: "****.ipa",
	signing_identity: "*************",
	bundle_version: "7.8.0.123456",
	short_version: "7.8.0",
	use_app_entitlements: true,
	provisioning_profile: {
           	"*.*.*" => path1,
		"com.*.*.plutoTodayWidgetST" => path2,
		"com.*.*.notificationServiceExtension" => path3
        }
	)

**Comment by minuscorp** [comment]
What about changing bitcode configuration? It usually throw errors like this one.

**Reply by zhoujun951236** [reply]
Our builds take over 10 minutes each. Running gym 3 times (dev, adhoc, appstore) is too long for a CI system. I'd love to just

match development
gym
match adhoc
resign
match appstore
resign

thk.What does this matter to bitcode?

**Reply by minuscorp** [reply]
It might affect to signing in some cases.

You can check out the docs to see if there's something that you might be missing out: https://docs.fastlane.tools/actions/sigh/#resign

**Reply by zhoujun951236** [reply]
hello,How can I be sure it has changed.I checked this document and found no omissions.
